compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  db:
+    image: postgres:17.2-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5433:5432"
```

Comment:
The database port is mapped from 5432 inside the container to 5433 on the host. Ensure that this port is not already in use on the host machine to avoid conflicts.